projects/tests/babel.config.js

```diff
@@ -0,0 +1,14 @@
+module.exports = {
```

Comment:
Should we source from the root babel config first, then overwrite as needed?:
```
{
   "extends": "../../babel.config.json",
   ...whatever here..
}
```
( #239 converts the babel config to json )

Reply:
added this!